### PR TITLE
Update to use shadow-cljs

### PR DESCRIPTION
**Pull request by madis**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  test:
+    working_directory: ~/ci
+    docker:
+      # Primary container image where all steps run.
+      - image: 487920318758.dkr.ecr.us-west-2.amazonaws.com/cljs-web3-ci:latest
+        aws_auth:
+            aws_access_key_id: $AWS_ACCESS_KEY_ID
+            aws_secret_access_key: $AWS_SECRET_ACCESS_KEY
+      # Secondary container image on common network.
+      - image: ipfs/go-ipfs
```
Comment (ginesdt) [inline .circleci/config.yml]: This is not related to ipfs, is this image needed?

File: shadow-cljs.edn [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{:dev-http {6502 {:root "out/browser-tests" :host "d0x-vm"}}
+ :nrepl {:port 30333}
+ :deps {:aliases [:dev]}
+ :builds
+ {:test-browser {:target :browser-test
```
Comment (ginesdt) [inline shadow-cljs.edn:5]: There is no tests in this lib. This is a bit misleading.

**Review by ginesdt — CHANGES_REQUESTED** [review]
This lib does not have unit tests. However, shadow config and circleci refers to tests, which might be a bit misleading. I suggest to either remove tests refs, or create some dummy test, some sort of placeholder, in case we want to add some tests in the future and comply with the config.

File: .circleci/config.yml (hunk 1 above)
Comment (madis) [reply .circleci/config.yml]: Nope. Removed along with adding a placeholder test

**Review by ginesdt — APPROVED** [review]
